src/libs/actions/PersonalDetails.js

```diff
@@ -284,7 +284,8 @@ function updateAvatar(file) {
  * Replaces the user's avatar image with a default avatar
  */
 function deleteAvatar() {
-    const defaultAvatar = ReportUtils.getDefaultAvatar(currentUserEmail);
+    // we want to use the old dot avatar here as this affects both platforms.
```

Comment:
```suggestion
    // We want to use the old dot avatar here as this affects both platforms.
```

Reply:
Also just wondering, are we planning to update OldDot to show all of these new icons? Or are we only going to get rid of `getOldDotDefaultAvatar` after reunification?